app/views/dashboards/following_tags.html.erb

```diff
@@ -22,7 +22,7 @@
             <span class="crayons-indicator crayons-indicator--outlined crayons-indicator--accent ml-3">Default 1.0</span>
         </div>
 
-        <%= form_with url: bulk_update_follows_path, method: :patch, local: true, id: "follows_update_form" do |f| %>
+        <%= form_with url: bulk_update_follows_path, method: :patch, local: true, class: "sticky bg-base-10 pt-3 pb-1 z-elevate", html: { style: "top: 55px;margin-left:-1px"}, id: "follows_update_form" do |f| %>
```

Comment:
UI (blocking) As per #12129, @ludwiczakpawel proposed to make the button sticky as you have done, but on the bottom instead exactly like on the http://localhost:3000/settings/customization page.

You can move this snippet of code below the list of tags and modify it like so.

```erb
        <%= form_with url: bulk_update_follows_path, method: :patch, local: true, class: "sticky bg-base-10 pt-3 pb-1 z-elevate", html: { style: "bottom: 0;margin-left:-1px"}, id: "follows_update_form" do |f| %>
          <button type="submit" class="crayons-btn crayons-btn--m mb-3" name="commit">Update Weights</button>
        <% end %>
```

Reply:
@nickytonline finally got around to implementing this. Looks great!

Reply:
Looks really good. Great stuff!